ios/PurchasesHybridCommon/PurchasesHybridCommon/RCPurchases+HybridAdditions.m

```diff
@@ -32,10 +32,26 @@ + (instancetype)configureWithAPIKey:(NSString *)APIKey
                         observerMode:observerMode
                         userDefaults:userDefaults
                       platformFlavor:platformFlavor
-               platformFlavorVersion:platformFlavorVersion];
+               platformFlavorVersion:platformFlavorVersion
+                   dangerousSettings:nil];
```

Comment:
This method no longer existed, but it wasn't seen at compile time because of the `Wincomplete-implementation` hack above.